src/content/app/genome-browser/components/drawer/drawer-views/variant-summary/VariantSummary.scss

```diff
@@ -55,9 +55,18 @@ For the reference (from Andrea)
   max-width: 60ch;
 }
 
-.clinicalSignificance {
-  display: flex;
-  flex-direction: column;
+.phenotypeAssociations {
+  margin-top: 15px;
+
+  .value {
+    display: flex;
+    column-gap: 10px;
+  }
+
+  svg {
+    fill: $green;
+    width: 12px;
+  }
```

Comment:
It works fine when there is a CADD line ([for rs699](http://update-clinical-sig-field.review.ensembl.org/genome-browser/grch38?focus=variant:rs699&location=1:230710029-230710068)) :

<img width="1440" alt="image" src="https://github.com/Ensembl/ensembl-client/assets/6834224/2ddcb165-94da-496e-9bcc-41e77aa61ce1">

but not so great for variants that don't have a CADD ([example link](http://update-clinical-sig-field.review.ensembl.org/genome-browser/grch38?focus=variant:rs779377781&location=13:32357725-32357764)):

<img width="721" alt="image" src="https://github.com/Ensembl/ensembl-client/assets/6834224/3e0d2768-1e1d-49f4-b15b-9ff52cf21547">


I suppose, a margin-bottom of 30px should help (it should collapse with the margin-top of CADD). Or perhaps both CADD and GERD will have to be wrapped in a container.